flow/templates/base_status.jinja

```diff
@@ -53,6 +56,19 @@
 {% endif %}
 {% endfor %}
 {% endfor %}
+
+{{ 'Detailed Aggregate View: \n' }}
+| operation | aggregate |
+| --------- | --------- |
```

Comment:
This has two sections and the table has three this leads to incorrect output. We need something like `|operation | aggregate | status`